tests/functional.rs

```diff
@@ -35,6 +35,8 @@ use farcaster_node::syncerd::types::{
     WatchAddress, WatchHeight, WatchTransaction,
 };
 
+use amplify;
```

Comment:
Do we need amplify if we don't use `s!` macro?

Reply:
No, removed it.